### ISSUE TITLE
`Homey.api(...)` (app settings) fails when using PUT/POST and `data` arguments is `null`

### ISSUE THREAD

**Issue by robertklep** [athombv/apps-sdk-issues#276]
This seems to happen on the mobile app only, not on the developer website.

This request throws an error (`Error: invalid_json`):
```
Homey.api('PUT', '/some/endpoint', null, callback);
```

When using `{}` instead of `null`, it works.

This contradicts [the documentation](https://apps.developer.homey.app/advanced/custom-views/app-settings#homey.api-string-method-string-path-mixed-body-function-callback) which states that _"The third argument body is optional, and null can be provided to ignore it"_

**Comment by robertklep** [comment]
FWIW, this also fails (again only in the mobile app):
```
Homey.api('POST', '/some/endpoint', callback);
```

**Comment by regnerus** [comment]
Hi @robertklep thanks for reporting! This issue will be fixed in release 6.11.12 of the mobile app.
